doc/tools/docgram/common.edit_mlg

```diff
@@ -0,0 +1,206 @@
+(* Defines additional productions and edits for use in documentation.  Not compiled into Coq *)
+
+DOC_GRAMMAR
+
+(* additional nts to be spliced *)
+
+LEFTQMARK: [
+| "?"
+]
+
+hyp: [
+| var
+]
+
+tactic_then_gen: [
+| EDIT ADD_OPT tactic_expr5 "|" tactic_then_gen
+| EDIT ADD_OPT tactic_expr5 ".." tactic_then_last
+]
+
+SPLICE: [
+| LEFTQMARK
+]
+
+SPLICE: [
+| hyp
+| identref
+| pattern_ident (* depends on previous LEFTQMARK splice todo: improve *)
+| constr_eval  (* splices as multiple prods *)
+| tactic_then_last  (* todo: dependency on c.edit_mlg edit?? really useful? *)
+| Prim.name
+| ltac_selector
+| Constr.ident
+| tactic_then_locality  (* todo: cleanup *)
+]
+
+RENAME: [
+  (* map missing names for rhs *)
+| _binders binders
+| Constr.constr constr

```

Comment:
Should become `term`, not `constr`.

Reply:
True.  There are quite a few other places where a similar change is needed in the output.  I was going to make that change as part of the Ltac changes PR but I can do it here.

Keep in mind the doc_grammar outputs are meant to illustrate the operation of the tool.  They are by no means the final version we'll use in the doc.